docs/docs/training-data-format.mdx

```diff
@@ -381,11 +381,11 @@ stories:
 Each step can be one of the following:
 
   - A [user message](#user-messages), represented by **intent** and **entities**.
```

Comment:
```suggestion
  - A [user message](#user-messages), represented by **intent** and **entities** or by a text.
```

Reply:
this is left the same intentionally, see discussion above